dev_guide/daemonsets.adoc

```diff
@@ -19,8 +19,36 @@ A daemonset can be used to run replicas of a pod on specific or all nodes in an
 Use daemonsets to create shared storage, run a logging pod on every node in
 your cluster, or deploy a monitoring agent on every node.
 
+For security reasons and by their nature creating DaemonSets is restricted to cluster administrators.
+
 For more information on daemonsets, see the link:http://kubernetes.io/docs/admin/daemons/[Kubernetes documentation].
 
+[IMPORTANT]
+====
+Before you start using daemonset you need to make sure to disable project default node selector in your namespace.
+
+If you are creating a new project make sure to overwrite the default node selector using
+`oc adm new-project --node-selector=""`.
+
+To disable the default node selector on existing project set namespace annotation
+`openshift.io/node-selector` to empty string:
+
+----
+# oc patch namespace myproject -p \
+    '{"metadata": {"annotations": {"openshift.io/node-selector": ""}}}'
+----
+
+xref:../admin_guide/managing_projects.adoc#using-node-selectors[Learn more about node selectors.]
+
+Daemonset scheduling is incompatible with project default node selector. Nor you usually want the default node selector
```

Comment:
Daemonset

Reply:
lowercase to honor rest of the doc, plus D in the start of a sentence